lld/test/COFF/arm64x-import.test

```diff
@@ -0,0 +1,533 @@
+REQUIRES: aarch64
```

Comment:
This file seems to require `x86` too, as we're disassembling some x86 thunks(?), see https://lab.llvm.org/buildbot/#/builders/190/builds/13515/steps/6/logs/FAIL__lld__arm64x-import_test.

Reply:
Yes, sorry about that. I pushed a fix as 80ab237c1187aa7e8a1f546175887d768fa14e2d.